contributors/devel/security-release-process.md

```diff
@@ -58,7 +58,37 @@ These steps should be completed within the first 24 hours of Disclosure.
 These steps should be completed within the 1-7 days of Disclosure.
 
 - The Fix Lead and the Fix Team will create a [CVSS](https://www.first.org/cvss/specification-document) using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). The Fix Lead makes the final call on the calculated CVSS; it is better to move quickly than make the CVSS prefect.
-- The Fix Team will notify the Fix Lead that work on the fix branch is complete once there are LGTMs on all commits in the private repo from one or more relevant assignees in the relevant OWNERS file.
+- To encourage fast adoption of the security fix, the new patched release should
+  contain only security fixes.
+- To facilitate this goal, the Fix Lead will [create a patch release branch](https://github.com/kubernetes/release/blob/master/docs/branching.md#branching-from-a-tag)
+  in a private fork of the public kubernetes.git repository, branched from the
+  most recent released tag for the release being fixed.
+  - As a hypothetical example, if `v2.4.7` is the latest public patch release for
+   `v2.4`, a new branch will be created in the private fork called
+   `release-2.4.7`, branching at the `v2.4.7` tag. If `HEAD` were to be used
+   instead, the security release might include non-security fixes.
+  - TODO: continuous integration testing should be set up for the branch
```

Comment:
My hope is that at some point the test infrastructure is sophisticated enough that we just tell it which branches we want to test, and it sets up a predetermined set of jobs against that branch. 

(This would be mainly used for the public release branches, but we should also be able to set it up for private branches.)

Setting up this test-infra would likely again reveal the presence of a security vulnerability (since the test configs are all public), but hopefully we would have disclosed its existence already (as previously noted).